.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,4 +26,4 @@
 - [ ] I have made corresponding changes to the documentation
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] If I included new strings, I have used `trans.` to make them localizable.
-      For more information see our [translations guide](https://napari.org/guides/stable/translations.html).
+      For more information see our [translations guide](https://napari.org/docs/dev/guides/translations.html).
```

Comment:
Ah we actually I think want to leave this as stable, but maybe the new page isn't there yet. @kne42 might need to manually copy it over for now so it's there before the next release, but we should have that automated soon.

Reply:
Would it be ok to leave it as it is for now (since it exists there). Also m ost of the user contributions are aimned at master, so it makes sense they have that up to date.

Reply:
It's a little more tricky as https://napari.org/docs/dev/guides/ is a slightly different site than https://napari.org/guides/stable/index.html (the navigation bar on the left is different), but it's probably strictly better to have the link not 404, so i'm ok to merge this now. We'll just want to fix later